gitops/argocd/tekton-chains/base/allow-argocd-to-manage.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: grant-argocd
+  namespace: tekton-chains
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+  - kind: ServiceAccount
+    name: argocd-sa-name  # Must be overriden in overlay
```

Comment:
the reason to override this value in overlay is to let users know that this is the way to do it? Or is there other rationale here?

Reply:
We might need a different value when we implement k8s support.